<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InteriorLights</class>
 <widget class="QDialog" name="InteriorLights">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="frontCheckbox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>96</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Front</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="midCheckbox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>96</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Mid</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="backCheckbox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>96</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
